emmalloc/emmalloc.c

```diff
@@ -56,6 +56,7 @@
 
 // Defind by the linker to have the address of the start of the heap.
 extern unsigned char __heap_base;
+extern unsigned char __heap_end __attribute__((__weak__));
```

Comment:
Since #394, this should no longer be weak.

In fact, building with `MALLOC_IMPL=emmalloc` is currently broken due to missing this symbol.